pocs/camera/camera.py

```diff
@@ -251,9 +353,105 @@ def get_thumbnail(self, seconds, file_path, thumbnail_size, keep_file=False, *ar
         image = fits.getdata(file_path)
         if not keep_file:
             os.unlink(file_path)
-        thumbnail = images.crop_data(image, box_width=thumbnail_size)
+        thumbnail = img_utils.crop_data(image, box_width=thumbnail_size)
         return thumbnail
 
+    def _fits_header(self, seconds, dark=None):
+        header = fits.Header()
+        if isinstance(seconds, u.Quantity):
+            seconds = seconds.to(u.second)
+            seconds = seconds.value
+        header.set('INSTRUME', self.uid, 'Camera serial number')
+        header.set('DATE-OBS', Time.now().fits)
```

Comment:
This doesn't look right with the `.fits` at the end.

Reply:
It's a bit ugly. Better as
```
now = Time.now()
header.set('DATE-OBS', now.fits)
```
?